File: easy/SmallestEvenMultiple.cs
namespace Easy;
/// <summary>
/// 
/// </summary>
public class SmallestEvenMultiple
{
    [Challenge("Smallest Even Multiple", 2413)]
    /// <summary>
    /// 
    /// </summary>
    /// <param name="n"></param>
    /// <returns></returns>

    public static int Solve(int n)
    {
        if (n % 2 != 0) return n * 2;
        return n;
    }
}

File: src/main/php/Easy/SmallestEvenMultiple.php
<?php

declare(strict_types=1);

namespace Src\Easy;

class SmallestEvenMultiple
{
    public function solve(int $n): int
    {
        if ($n % 2 !== 0) {
            return $n * 2;
        }

        return $n;
    }
}
